__tests__/app.test.js

```diff
@@ -111,51 +111,133 @@ describe("/api/articles/:article_id", () => {
 });
 
 describe("/api/articles/:article_id/comments", () => {
-  test("GET 200: Responds with an array of comments for the given article with the appropriate properties", () => {
-    return request(app)
-      .get("/api/articles/1/comments")
-      .expect(200)
-      .then(({ body }) => {
-        const { comments } = body;
-        expect(comments.length).toBe(11);
-        comments.forEach((comment) => {
-          expect(comment).toMatchObject({
-            comment_id: expect.any(Number),
-            votes: expect.any(Number),
-            created_at: expect.any(String),
-            author: expect.any(String),
-            body: expect.any(String),
-            article_id: expect.any(Number),
+  describe("GET", () => {
+    test("GET 200: Responds with an array of comments for the given article with the appropriate properties", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
           });
         });
-      });
-  });
-  test("GET 200: Comments should be served by order of most recent to oldest",()=>{
-    return request(app)
-    .get("/api/articles/1/comments")
-    .expect(200)
-    .then(({body})=>{
-        const {comments} = body
-        expect(comments).toBeSortedBy("created_at", { descending: true })
-    })
-  })
-  test("GET 400: Invalid article_id type received", () => {
-    return request(app)
-      .get("/api/articles/invalid-input/comments")
-      .expect(400)
-      .then(({ body }) => {
-        const { msg } = body;
-        expect(msg).toBe("Bad request");
-      });
+    });
+    test("GET 200: Comments should be served by order of most recent to oldest", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+    test("GET 400: Invalid article_id type received", () => {
+      return request(app)
+        .get("/api/articles/invalid-input/comments")
+        .expect(400)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toBe("Bad request");
+        });
+    });
+    test("GET 404: article_id comments not found", () => {
+      return request(app)
+        .get("/api/articles/2/comments")
+        .expect(404)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toBe("Not found");
+        });
+    });
```

Comment:
I'd expect to see a 200 test for an article with no comments. It isn't an error for an article to not have any associated comments. A brand new article wouldn't have any because no one would have read it yet, that wouldn't be an issue.